keyboards/opticalkb/readme.md

```diff
@@ -0,0 +1,50 @@
+# optkb
+
+*A short description of the keyboard/project*
+
+* Keyboard Maintainer: [Girish Palya](https://github.com/girishji)
+* Hardware Supported: Black pill STM32F401
+
+PCB for this keyboard can be found on [github](https://github.com/girishji/optical-keyboard).
+
+Make example for this keyboard (after setting up your build environment):
+
+    make opticalkb:default
+
+Flashing example for this keyboard:
+
+    make opticalkb:default:flash
+
+See the [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) and the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information. Brand new to QMK? Start with our [Complete Newbs Guide](https://docs.qmk.fm/#/newbs).
+
+## Bootloader
+
+Enter the bootloader in 3 ways:
+
+* **Physical reset button**: Briefly press the button BOOT and then button NRST, release NRST first and then BOOT 
+* **Keycode in layout**: Press the key mapped to `RESET` if it is available
+
+Verify that it entered DFU (bootloader) mode through QMK Toolbox before
+flashing.
+
+## Wiring instructions
+
+There is only one way to solder Blackpill (STM32F401) to the [PCB](https://github.com/girishji/optical-keyboard).
+
+#### Notes
+
+- On Blackpill boards, **avoid** using the following pins, since they will cause either USB enumeration or the DFU bootloader to not work correctly:
```

Comment:
This is linked in the docs:
https://docs.qmk.fm/#/platformdev_blackpill_f411

Reply:
Thank you